sdcm/teardown_validators/events.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import re
+
+from sdcm.sct_events import Severity
+from sdcm.sct_events.file_logger import get_events_grouped_by_category
+from sdcm.teardown_validators.base import TeardownValidator
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ErrorEventsValidator(TeardownValidator):  # pylint: disable=too-few-public-methods
+    validator_name = 'error_events'
+
+    def validate(self):
+        # Do not run validator if there were no parallel nemeses
+        if len(self.tester.get_nemesis_class()) == 1:
```

Comment:
Let's leave for SCT configuration to decide which case to enable it

No need to enforce it here